config/default/manager_auth_proxy_patch.yaml

```diff
@@ -15,8 +15,9 @@ spec:
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.15.0
         args:
+        - "--http2-disable"
```

Comment:
I guess it's still not safe to explain why we're adding `--http2-disable`? :disappointed: